### ISSUE TITLE
Search function not working with many open tabs

### ISSUE THREAD

**Issue by rakoczy** [2alin/tabFx#4]
Search function stops working after some time if there are many (about hundred) tabs open (in a dozen of windows). The effect is reversible - when I closed most tabs and restarted Firefox tabFx started to work.
Firefox 63.0
Windows 7

**Comment by 2alin** [comment]
Thank you for reporting it. I'll try to reproduce such behavior and see how to fix it.

**Comment by 2alin** [comment]
I tested it already with 108 tabs (wiki pages, news websites, youtube channels, etc) and didn't have any crash with the extension or the browser. But what I noticed was a certain delay (1-2 seconds) in the synchronization of the filtered list and the anchors to the respective tabs (check gif at the end). I'll check that on the next addon update. Also, I need to fix the favicon problem.

![screencast 2018-11-01 13 27 35](https://user-images.githubusercontent.com/4089035/47875491-4bdb3180-dddc-11e8-9bec-b7258d752cb2.gif)
